front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Origin.tsx

```diff
@@ -5,36 +5,43 @@ import { RiMapPin2Fill } from 'react-icons/ri';
 import { BiLink, BiUnlink } from 'react-icons/bi';
 import { useTranslation } from 'react-i18next';
 
-import {
-  updateOrigin,
-  updateOriginDate,
-  updateOriginTime,
-  updateOriginUpperBoundDate,
-  updateOriginUpperBoundTime,
-  toggleOriginLinkedBounds,
-  updatePathfindingID,
-} from 'reducers/osrdconf';
-import {
-  getMode,
-  getOrigin,
-  getOriginDate,
-  getOriginTime,
-  getOriginLinkedBounds,
-  getOriginUpperBoundDate,
-  getOriginUpperBoundTime,
-} from 'reducers/osrdconf/selectors';
 import { makeEnumBooleans } from 'utils/constants';
 
 import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
 import { MODES } from 'applications/operationalStudies/consts';
+import { useOsrdActions, useOsrdContext, useOsrdSelectors } from 'common/osrdContext';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done